package-overrides/github/tagit/tagit@^1.11.4.json

```diff
@@ -0,0 +1,11 @@
+{
+  "registry": "jspm", 
+  "main": "js/tag-it", 
+  "shim": { "js/tag-it": 
+  { "deps": ["jquery","jquery-ui","../css/jquery.tagit.css!"],
+     "exports":"$" } }, 
+  "dependencies": { "jquery": "github:components/jquery",
+                 "jquery-ui":"github:components/jqueryui@1.11.4",
```

Comment:
would it be okay if i replace those dependencies lines with  
"dependencies": { "jquery": "github:components/jquery",
                              "jquery-ui":"jspm:jqueryui@^1.8"